lib-src/portmixer/include/portmixer.h

```diff
@@ -57,7 +57,7 @@ typedef float PxBalance; /* -1.0 (left) --> 1.0 (right) */
  audio device.  Pass 0 as the index for the first (default) mixer.
 */
 
-PxMixer *Px_OpenMixer( void *pa_stream, int i );
+PxMixer* Px_OpenMixer(PaStream* pa_stream, PaDeviceIndex inputDeviceIndex, PaDeviceIndex outputDeviceIndex, int i);
```

Comment:
I see `int i` is always zero in existing calls.  That doesn't necessarily mean the argument should be removed.  But I wonder if the generality will ever be needed, or has been tested.

Reply:
I notice new argument names are camelCase but the library otherwise prefers snake_case for variable and member names.  Not a serious matter, just style.